### ISSUE TITLE
Possible UBL autolevel bug ?

### ISSUE THREAD

**Issue by Jens5395** [MarlinFirmware/Marlin#11775]
I am currently runnin Marlin 1.1.9 (release) on a CR10-S printer
I have configured for a BLTouch probe
Printbed is defined as 300 * 300
Mesh inset is 0, Mesh min x and y is 0, mesh max x and y is 300
probe offset is 0/0 for x and y. In other words I am telling the printer that the probe is at the same position as the extruder
I have a 5 point by 5 point auto probing grid defined
I am expecting the printer to probe the following spots:
x0, x75, x 150. x 225 and x300
Y locations should be identical

What I get instead is 
x75, x150,and x225 and the same y positions

In other words, the printer never probes the extreme x and y points
This used to work under Marlin 1.1.8

Now maybe there is a reason for this behaviour that I do not understand ?
Or maybe it's a bug ?

I looked over the open issues but did not find anything matching this error
I do not (yet) know where to find the 1.1.9 bugfix issue to test (sorry)

**Comment by ManuelMcLure** [comment]
Check your setting for `MIN_PROBE_EDGE` - it defaults to 10mm to handle inductive and capacitive probes that do not behave well at the very edge of the bed. If your probe behaves correctly at the edges of the bed you can set it to 0.

**Comment by Jens5395** [comment]
Man do I feel stupid :(   ...... YES, that was exactly what the problem was!  
THANK YOU !!!!!   I was screwing around with that for the better part of 10 hours I believe .... sad but true ...
